terraform/environments/main.tf

```diff
@@ -3,7 +3,7 @@ variable "vault_addr" {
 }
 
 module "aws_deploy-main-ap-southeast-1" {
-  source            = "../modules/cloud/aws/deploy"
+  source            = "git::https://github.com/aeternity/terraform-aws-aenode-deploy.git"
```

Comment:
How is versioning handled? 🤔

Reply:
I mean - commit in this repo infrastructure shall link to immutable reference to the terraform-aws-aenode-deploy repo. And I see no version / tag / commit hash of terraform-aws-aenode-deploy in this line.

Reply:
It defaults to HEAD, @valerifilipov perhaps we should tag module v1.0.0 (as it's production tested already) and reference it here ?

Reply:
Also why not prefer the short version `github.com/aeternity/terraform-aws-aenode-deploy` ?

Reply:
Yes, we should tag both repos, after https://github.com/aeternity/terraform-aws-aenode-deploy/pull/1 and https://github.com/aeternity/terraform-aws-api-gateway/pull/1 get merged

Reply:
You don't need CI integrations to tag it